ceph-releases/ALL/ubuntu/__DOCKERFILE_POSTINSTALL_CLEANUP__

```diff
@@ -1,6 +1,5 @@
 echo 'Postinstall cleanup' && \
      ( echo "apt clean" && DEBIAN_FRONTEND=noninteractive apt-get clean && \
-      echo "apt autoremove" && DEBIAN_FRONTEND=noninteractive apt-get autoremove && \
```

Comment:
Why remove the `autoremove` command. All this does is remove dependencies that are no longer needed. It shouldn't break things that were explicitly installed. But, for example, it will remove the unneeded dependencies pulled in by `wget` if `wget` is removed.

Reply:
in fact it did removed everything, including ceph ...

Reply:
Huh ... that's not good. I wouldn't think that should happen, but obviously it has to be removed.